proxy/src/main/java/com/wavefront/agent/handlers/ReportableEntityHandlerFactoryImpl.java

```diff
@@ -20,7 +20,7 @@
   private static final Logger logger = Logger.getLogger(
       ReportableEntityHandlerFactoryImpl.class.getCanonicalName());
 
-  private static final int SOURCE_TAGS_NUM_THREADS = 2;
+  private static final int API_NUM_THREADS = 2;
```

Comment:
possible keep SOURCE_TAG_NUM_THREADS and add EVENT_NUM_THREADS

Reply:
done